docs/implementing-cards.md

```diff
@@ -799,45 +799,51 @@ Some card abilities require multiple targets. These may be specified using the `
 
 Once all targets are chosen, they will be set using their specified name under the `targetResolvers` property on the handler context object.
 
-<!-- TODO: update this section -->
-#### **IGNORE FOR NOW, WIP:** ~~Select options~~
 
-Some abilities require the player (or their opponent) to choose between multiple options.  This is done in the same way as targets above, but by using the `mode` property set to `'select'`.  In addition, a `choices` object should be included, which contains key:value pairs where the key is the option to display to the player, and the value is either a function which takes the `context` object and returns a boolean indicating whether this option is legal, or a game action which will be evaluated on the basis of the specified target (or default as detailed below) to determine whether the choice is legal.  The selected option is stored in `context.select.choice` (or `context.selects[targetName].choice` for an ability with multiple targets).
+#### Select options
```

Comment:
Just a heads-up, I marked at the top of these md files but we're transitioning over to the wiki. Do you mind making these same updates over there?